src/library_legacy.js

```diff
@@ -44,6 +44,7 @@ mergeInto(LibraryManager.library, {
   // in a maximum length that can be used to be secure from out of bounds
   // writes.
   $writeStringToMemory__docs: '/** @deprecated @param {boolean=} dontAddNull */',
+  $writeStringToMemory__deps: ['$lengthBytesUTF8', '$stringToUTF8Array'],
```

Comment:
```suggestion
  $writeStringToMemory__deps: ['$lengthBytesUTF8', '$stringToUTF8'],
```

Reply:
The comment on line 53 looks out of date, as the code on line 59 uses the non-array version.